docs/using-the-jdbc-driver/using-plugins/UsingTheDeveloperPlugin.md

```diff
@@ -0,0 +1,54 @@
+## Developer Plugin
+
+> :warning: The plugin is NOT intended to be used in production environments. It's designed for the purpose of testing.
+
+The Developer Plugin allows developers to inject an exception to a connection and to verify how an application handles it. 
+
+Since some exceptions raised by the drivers happen rarely, testing for those might be difficult and require a lot of effort in building a testing environment. Exceptions associated with network outages are good example of those exceptions. It may require substantial efforts to design and to build a testing environment where such timeout exceptions could be produced with 100% accuracy and 100% guarantee. If test suite can't produce and verify such network outages cases with 100% accuracy it significantly decreases the value of such tests and makes the tests unstable and flaky. The Developer Plugins simplifies testing of such scenarios as shown below.
```

Comment:
```suggestion
Since some exceptions raised by the drivers rarely happen, testing for those might be difficult and require a lot of effort in building a testing environment. Exceptions associated with network outages are a good example of those exceptions. It may require substantial efforts to design and build a testing environment where such timeout exceptions could be produced with 100% accuracy and 100% guarantee. If a test suite can't produce and verify such cases with 100% accuracy it significantly decreases the value of such tests and makes the tests unstable and flaky. The Developer Plugin simplifies testing of such scenarios as shown below.
```
?